### PR TITLE
ref(sentry): Ignore some fetch/xhr breadcrumbs

### PR DESCRIPTION
**Pull request by evanpurkhiser**
These just end up polluting our breadcrumbs

### CODE REVIEW

File: static/app/bootstrap/initializeSdk.tsx [hunk 1]
```diff
@@ -19,6 +19,12 @@ const SPA_MODE_ALLOW_URLS = [
   'webpack-internal://',
 ];
 
+// We don't care about recording breadcrumbs for these hosts. These typically
+// pollute our breadcrumbs since they may occur a LOT.
+//
+// XXX(epurkhiser): Note some of these hosts may only apply to sentry.io.
+const INGORED_BREADCRUMB_FETCH_HOSTS = ['aplitude.com', 'reload.getsentry.net'];
```
Comment (priscilawebdev) [inline static/app/bootstrap/initializeSdk.tsx:26]: didn't you mean `amplitude?

Comment (priscilawebdev) [reply static/app/bootstrap/initializeSdk.tsx:26]: and if our customers are using it?

File: static/app/bootstrap/initializeSdk.tsx [hunk 2]
```diff
@@ -144,6 +150,21 @@ export function initializeSdk(config: Config, {routes}: {routes?: Function} = {}
       if (isFilteredRequestErrorEvent(event) || isEventWithFileUrl(event)) {
         return null;
       }
+
+      event.breadcrumbs = event.breadcrumbs?.filter(crumb => {
```
Comment (lforst) [inline static/app/bootstrap/initializeSdk.tsx:154]: In theory we have a `beforeBreadcrumb` hook that works similar to `beforeSend` but is more isolated to breadcrumbs. It is probably preferrable to this logic. If you wanna drop a breadcrumb you can simply return `null` in the `beforeBreadcrumb` hook.

Comment (evanpurkhiser) [reply static/app/bootstrap/initializeSdk.tsx:154]: oh nice, even better

Comment (priscilawebdev) [reply static/app/bootstrap/initializeSdk.tsx:154]: agree that it is better than handling this in the UI

File: static/app/bootstrap/initializeSdk.tsx (hunk 1 above)
Comment (evanpurkhiser) [reply static/app/bootstrap/initializeSdk.tsx:26]: This is our instance our the sdk

Comment (priscilawebdev) [reply static/app/bootstrap/initializeSdk.tsx:26]: oh yes..I just saw it :)

File: static/app/bootstrap/initializeSdk.tsx (hunk 2 above)
Comment (evanpurkhiser) [reply static/app/bootstrap/initializeSdk.tsx]: Updated! much cleaner, thanks for jumping in on this @lforst

**Review by k-fish — APPROVED** [review]
Let's do it. I removed similar bits for spans because of the noise and impact on transaction duration.

**Pull request merged**